README.md

```diff
@@ -30,6 +30,27 @@ There are plenty of small, self-contained examples [in the documentation][exampl
 
 The easiest way to get started using ct is to download one of the binaries from the [releases page on GitHub][releases].
 
+One can use the following script to download and verify the signature of Config Transpiler:
+
+```bash
+# Sepcify Config Transpiler version
+CT_VER=v0.6.1
+# Specify download URL
+DOWNLOAD_URL=https://github.com/coreos/container-linux-config-transpiler/releases/download
+
+# Download Config Transpiler binary
+curl -L ${DOWNLOAD_URL}/${CT_VER}/ct-${CT_VER}-x86_64-unknown-linux-gnu -o /tmp/ct-${CT_VER}-x86_64-unknown-linux-gnu
```

Comment:
AFAICT this should also work on macOS, maybe add `${OS}` like `CT_VER` and `DOWNLOAD_URL` and list the options as a comment? I don't have a mac to test this however.